### DISCUSSION TITLE
Add site to multiple tenants

### DISCUSSION THREAD

**Discussion by GeertReijnders** [netbox-community/netbox#17914] [Ideas]
Hi,
We have multiple customers on the same physicall address. However, these customers could also have another office on a different address. How can I group this, so it is easy to see on which address customers have offices and thus devices, ip spaces, circuits and so on?
![multiple customers](https://github.com/user-attachments/assets/c5a644e6-5931-40a9-878b-876a66269921)
I hope this pictures makes sense :)

Thanks.

**Comment by ghost** [comment]
tags are your answer here. you can do it with custom fields too, but tags offer easier one-click filtering.

Similar functionality has been proposed a few times:
https://github.com/netbox-community/netbox/issues/17123
https://github.com/netbox-community/netbox/issues/10784

In https://github.com/netbox-community/netbox/issues/10784 what you're looking for has been declined to be implemented.

What we do is create tags with prefixes, specifically "bu:" for business unit. Then we tag sites with the multiple business units that occupy that address.
